interface/patient_file/summary/demographics.php

```diff
@@ -428,7 +428,9 @@ function load_location(location) {
         placeHtml("pnotes_fragment.php", 'pnotes_ps_expand');
         placeHtml("disc_fragment.php", "disclosures_ps_expand");
         placeHtml("labdata_fragment.php", "labdata_ps_expand");
+        <?php if (file_exists($GLOBALS['file_root'] . "/interface/forms/track_anything/info.txt")) { ?>
         placeHtml("track_anything_fragment.php", "track_anything_ps_expand");
```

Comment:
I thought I removed this issue (note the link is basically skipped since track_anything_ps_expand element does not exist)

Reply:
really a double whammy. one, promise errors because div doesn't exist and two, null.innerHTML = fragment will still write to the form. where, god knows!
so why do the fetch if we don't need to. still function should be fixed...

Reply:
Agreed. It actually took me awhile to even figure out how the original demographics.php script did this (by not having the element there). The problem with the file_exists method is that it always exists. I did a hail mary search for track_anything in the codebase and turns out it is my lucky day (I almost felt like I wont a lottery; sure I didn't win any money, so I guess it's more like I won a game of bingo):
https://github.com/openemr/openemr/blob/master/interface/main/tabs/main.php#L233

Reply:
My only request is to not turn it into a session variable :)

Reply:
sorry @bradymiller I didn't see where you mod'ed fetchHtml. That works just fine. I was working 502 where I didn't have that mod.
I'll leave this open with fix in case I run across something else.